### ISSUE TITLE
Decreasing the width of the TailRecyclerView

### ISSUE THREAD

**Issue by linkinamine** [Ramotion/garland-view-android#6]
Is there a way to decrease the width of the completely visible element inside the `TailRecyclerView`, `itemStart` is not working and neither is  playing with the `TailRecyclerView`width.

**Comment by dvg4000** [comment]
It's a bug, I'll fix it soon.

**Comment by dvg4000** [comment]
Bug fixed in version `0.2.1`. 
You can use XML attributes like this:

```xml
<com.ramotion.garlandview.TailRecyclerView
    ...
    app:itemStart="50dp"
    .../>
```

Or use special  constructor:

```java
public class MainActivity extends AppCompatActivity implements GarlandApp.FakerReadyListener {
    ...

    private void initRecyclerView(List<List<InnerData>> data) {
        ...
        final TailRecyclerView rv = (TailRecyclerView) findViewById(R.id.recycler_view);
        final TailLayoutManager lm = new TailLayoutManager(100 /*itemStart in pixels*/, 0 /*itemGap in pixels*/);
        lm.setPageTransformer(new HeaderTransformer());
        rv.setLayoutManager(lm);
        ...
    }
```
